examples/v1beta1/trial-images/tf-mnist-with-summaries/mnist.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A simple MNIST classifier which displays summaries in TensorBoard.
+
+This is an unimpressive MNIST model, but it is a good example of using
+tf.name_scope to make a graph legible in the TensorBoard graph explorer, and of
+naming summary tags so that they are grouped meaningfully in TensorBoard.
+
+It demonstrates the functionality of every TensorBoard dashboard.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import tensorflow as tf
+
+# from tensorflow.examples.tutorials.mnist import input_data
+import input_data
```

Comment:
Can we re-use existing package, like here: https://github.com/kubeflow/training-operator/blob/master/examples/tensorflow/mnist_with_summaries/mnist_with_summaries.py#L31 ?
Then, we don't need to add input_data to our repo.

Reply:
We need to add `input_data.py` in our repository since It removed from tensorflow repository. 

Related: 
  - https://github.com/tensorflow/tensorflow/commit/42e122e6a0acaa7bee32ce88e2423ed99b3a0d84
  - https://github.com/tensorflow/tensorflow/commit/ad2c68c45fe13946c58d6646c56f308eee6856a1

Reply:
I see, thank you for letting me know.
Maybe we can try to refactor our example to re-use the default TF API to download dataset: https://www.tensorflow.org/tutorials/quickstart/beginner#load_a_dataset.
What do you think @tenzen-y ?

Reply:
> Maybe we can try to refactor our example to re-use the default TF API to download dataset

It makes sense.
I would like to refactor this example using [tensorflow-datasets](https://www.tensorflow.org/datasets/overview).

Reply:
Thank you!